bin/check-redis-list-length.rb

```diff
@@ -69,6 +69,12 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  option :conn_failure_status,
+         long: '--conn-failure-status EXIT_STATUS',
+         description: 'Exit status for Redis connection failures',
```

Comment:
This could be improved a bit to say "Returns the following exit status for Redis connection failures" which seems a bit more descriptive and does not me to jump to the code to understand what you mean.